tests/ci/run_cross_ppc64le_tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+SCRIPT_DIR="$(readlink -f "${SCRIPT_DIR}")"
+
+"${SCRIPT_DIR}/run_cross_tests.sh" ppc64le powerpc64le-unknown-linux-gnu "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_BUILD_TYPE=Release -DFIPS=1 -DBUILD_SHARED_LIBS=1"
```

Comment:
I'm seeing that the qemu simulation is pretty slow.. do we care enough to test other configuration types like non-FIPS or the debug build in separate CI jobs? This can be in another PR though.